resource_transip_vps_test.go

```diff
@@ -70,3 +68,33 @@ func TestAccTransipResourceVps(t *testing.T) {
 		},
 	})
 }
+
+func TestAccTransipResourceVpsUpdate(t *testing.T) {
+	if os.Getenv("THIS_IS_GOING_TO_COST_ME_MONEY") == "" {
+		t.Skip("THIS_IS_GOING_TO_COST_ME_MONEY not set, skipping")
+	}
+	timestamp := time.Now().Unix()
+	// TODO: Add tags when supported.
+	testConfig := fmt.Sprintf(`
+	resource "transip_vps" "test" {
+		description             = "test-%d"
+	product_name     = "vps-bladevps-x1"
+	operating_system = "ubuntu-20.04"
+	is_customer_locked = true
+	}
+	`, timestamp)
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
```

Comment:
Could you add a second step that does an Update as well, like changing the customer lock to false? This would thoroughly test the update functionality.

Reply:
I've added a second `testConfig` that changes the customer lock to false and updates the description